webrtc-c/canary/jobs/orchestrator.groovy

```diff
@@ -96,10 +117,23 @@ pipeline {
                             parameters: COMMON_PARAMS + [
                                 booleanParam(name: 'USE_TURN', value: true),
                                 booleanParam(name: 'TRICKLE_ICE', value: true),
-                                string(name: 'DURATION_IN_SECONDS', value: DURATION_IN_SECONDS.toString()),
-                                string(name: 'MASTER_NODE_LABEL', value: "personal"),
-                                string(name: 'VIEWER_NODE_LABEL', value: "personal"),
-                                string(name: 'RUNNER_LABEL', value: "Default"),
+                                string(name: 'DURATION_IN_SECONDS', value: PERIODIC_DURATION_IN_SECONDS.toString()),
+                                string(name: 'MASTER_NODE_LABEL', value: "ec2-us-west-2"),
```

Comment:
Is there a way to get change these values in run time? If we were to change the node, it would have to be manually right?

Reply:
Yep, we need to do it manually. But, I think if we have a strict naming convention, we won't update these labels that much.